homeassistant/components/hue/sensor_base.py

```diff
@@ -64,9 +68,14 @@ def __init__(self, bridge):
 
     async def async_register_component(self, platform, async_add_entities):
         """Register async_add_entities methods for components."""
+        if platform not in self._enabled_platforms:
+            _LOGGER.debug("Aborting %s. It is not enabled", platform)
+            return
```

Comment:
When can this happen ?

Reply:
Right now, never :)

I forget that when reverting the remote platform, I'll clean it